pipelines/scheduled/coverage/coverage.yml

```diff
@@ -5,7 +5,8 @@ steps:
         key:   "coverage"
         # Comment out the following `if:` statement for testing.
         # Please remember to uncomment it before merging the PR.
-        if: pipeline.slug == "julia-master-scheduled"
+        # if: build.source == "schedule" # comment out this line for testing; please uncomment it before merging your PR
+        if: (build.source == "schedule") || (pipeline.slug == "julia-buildkite") || (pipeline.slug == "julia-buildkite-scheduled") # uncomment this line for testing; please comment it out before merging your PR
```

Comment:
These lines are outdated now that we have moved to a separate scheduled pipeline.

Take a look at how we do it on master: https://github.com/JuliaCI/julia-buildkite/blob/main/pipelines/scheduled/coverage/coverage.yml

Roughly speaking, you want to delete the two `if` lines (i.e. delete lines 8 and 9) and replace them with the following three lines:

```yml
        # Comment out the following `if:` statement for testing.
        # Please remember to uncomment it before merging the PR.
        if: pipeline.slug == "julia-master-scheduled"
```